polymetis/polymetis/python/torchcontrol/policies/velocity.py

```diff
@@ -21,6 +21,7 @@ def __init__(
         Kp,
         Kd,
         robot_model: torch.nn.Module,
+        hz=torch.tensor(0, dtype=torch.int32),
```

Comment:
Since the constructor is being run on the python side, there is no need to constrain arguments to be torch tensors.
I would designate `hz` as a required int argument and remove the part where it is designated on the fly.

Also, since `self.hz` is not of type tensor, remember to annotate the attribute as explained in the [TorchScript docs](https://pytorch.org/docs/stable/jit.html#torchscript-classes).